libraries/AP_GPS/AP_GPS.h

```diff
@@ -533,6 +533,12 @@ class AP_GPS
     bool get_error_codes(uint8_t instance, uint32_t &error_codes) const;
     bool get_error_codes(uint32_t &error_codes) const { return get_error_codes(primary_instance, error_codes); }
 
+    enum class SBAS_Mode : int8_t {
```

Comment:
```suggestion
    enum class SBASMode : uint8_t {
```

Not commenting on which is better - but this is our pattern.  But hey, if you'd like to argue about this one with other people for way too long, by all means, leave it signed :-)

Reply:
Because the underlying store as `AP_Int8` I was just mirroring the actual type that mattered.... It doesn't actually change the build size either way though...